### PR TITLE
[RDY] More mixed alcohol recipes

### PR DESCRIPTION
**Pull request by angoddu**
I started this branch when I saw that mixed weak alcohol was completely useless. While I was tweaking weak alcohol I also improved strong alcohol. I made the following changes:

- Made weak mixed alcohol a component in fancy hobo. 
- Made strong mixed alcohol and ethanol components in tainted tornado, sewer brew, spiked eggnog, and fancy hobo. 
- Added denatured alcohol and ethanol components in improvised first aid kit. 
- Made new requirement groups for ease of coding.

What else could I do to make weak mixed alcohol more useful? Would it be at all possible to refine it into ethanol?

### CODE REVIEW

**Comment by Reclusive-reptile** [comment]
There is no reason weak mixed alcohol could not be distilled into ethanol. It should be a bout 5% ethanol by volume. So you would just need a lot of it to get concentrated alcohol out of it.

**Comment by angoddu** [comment]
I'll add that to the ethanol recipe. Would it also be possible to make denaturalized alcohol the same way?

**Comment by Reclusive-reptile** [comment]
I don't see why not. Denatured ethanol just has impurities that make it unpalatable or not food grade, and our survivor is making it via a "blind" distillation and would not be able to clear those impurities.

File: data/json/requirements/cooking_components.json [hunk 1]
```diff
@@ -12,6 +12,18 @@
     "components": [ [ [ "moonshine", 1 ], [ "vodka", 1 ], [ "whiskey", 1 ], [ "single_malt_whiskey", 1 ], [ "brandy", 1 ], [ "rum", 1 ], [ "gin", 1 ], [ "tequila", 1 ], [ "mixed_alcohol_strong", 1 ] ] ]
   },
   {
+    "id": "hard_liquor_chem",
+    "type": "requirement",
+    "//": "Hard liquor and ethonal - Must be diluted. Added for ease of use",
```
[the remark this answers is not in the document]
Comment (angoddu) [reply data/json/requirements/cooking_components.json]: Cool fact. I'll fix it

**Comment by angoddu** [comment]
Can someone tell me what is going wrong with this PR? I don't see any typos and I don't know if I have any style problems.

**Comment by Night-Pryanik** [comment]
> Can someone tell me what is going wrong with this PR? I don't see any typos and I don't know if I have any style problems.

Travis sometimes complain about what's-on-his-mind. That's most probably not your fault. If gorgon is saying OK, then your PR doesn't have any typos or styling errors.

**Comment by angoddu** [comment]
> Travis sometimes complain about what's-on-his-mind

So then is this safe to merge?

**Comment by Night-Pryanik** [comment]
> So then is this safe to merge?

I suppose so.

**Comment by Coolthulhu** [comment]
I re-ran the failed job. It should show as all checks passed now.

**Pull request merged**